Mathlib/Algebra/Category/ModuleCat/Monoidal/Symmetric.lean

```diff
@@ -29,6 +29,7 @@ namespace MonoidalCategory
 @[simp]
 theorem braiding_naturality {X₁ X₂ Y₁ Y₂ : ModuleCat.{u} R} (f : X₁ ⟶ Y₁) (g : X₂ ⟶ Y₂) :
     (f ⊗ g) ≫ (Y₁.braiding Y₂).hom = (X₁.braiding X₂).hom ≫ (g ⊗ f) := by
+  ext : 1
   apply TensorProduct.ext'
```

Comment:
We should really fix the issue that makes `TensorProduct.ext'` not apply using the `ext` tactic...